src/components/MainContent/TaskModel.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { RenderListItems, Stem, Task } from './index';
+import { Content } from './Content';
+import { MainHeader, NonBulletList, NumberList, Passage, Section, SubHeader } from './Components';
+
+interface TaskModelProps {
+  task: Task;
+  stems: Stem[];
+  evidences: string[];
+  scoringRule: string;
+  index: number;
+}
+
+export const TaskModel = ({ task, stems, evidences, scoringRule, index }: TaskModelProps) => {
+  const taskPrefix = `taskModel${index}`;
+  const desc = Content(task.taskDesc);
+
+  const RenderStemsBy = (target: string, stems: Stem[]) =>
+    stems.filter(stem => stem.shortStem === target).map((stem, index) => {
+      const desc = Content(stem.stemDesc);
+
+      return <li key={index}>{desc}</li>;
+    });
+
+  const RenderAppropriateStems = (stems: Stem[]) => RenderStemsBy('Appropriate Stems', stems);
+
+  const RenderDualTextOnlyStems = (stems: Stem[]) =>
+    RenderStemsBy('Appropriate Stems for Dual-Text Stimuli', stems);
+
+  const RenderFormatExample = (example: string, sectionName: string) => {
+    let elements: JSX.Element | undefined;
+
+    if (example === 'NA') return;
```

Comment:
I would add a comment or two here to this line and the next as to why you need to return after encountering these `example`s. Since I don't know the data very well, it's not immediately clear.